component-library/src/app/pages/banner-documentation/banner-doc-code.component.ts

```diff
@@ -403,6 +443,28 @@ export class BannerDocCodeComponent implements OnInit {
     };
   }
 
+  private parseCodeViewConfig() {
+    const index = this.codeViewConfig?.tab?.findIndex((t) => t.id === 'ts');
+    if (-1 == index || !index) return;
+    this.bannerConfigCodeView = {
+      ...this.bannerConfigCodeView,
+      id: this.bannerConfig.id,
+      title: this.bannerConfig.title,
+      content: this.bannerConfig.content,
+      type: this.bannerConfig.type,
+      rounded: this.bannerConfig.rounded,
+      dismissible: this.bannerConfig.dismissible,
+      cta: this.bannerConfig.cta,
+      size: this.bannerConfig.size,
+      ariaDissmissible: this.bannerConfig.ariaDissmissible
+    };
+    if (this.codeViewConfig?.tab) {
+      this.codeViewConfig.tab[index].value =
+        "import { IBannerConfig } from 'ircc-ds-angular-component-library';\n\n"+
```

Comment:
I went with this style because it makes the code and the string easier to format and indent, otherwise the indentation of the code is off and if you don't move the code back against the wall it adds extra spaces